lib/DDG/Spice/BiblioSearch/BooksByAuthor.pm

```diff
@@ -0,0 +1,53 @@
+package DDG::Spice::BiblioSearch::BooksByAuthor;
+# ABSTRACT: This module fetches books authored by a person searched for in DDG.
+
+use strict;
+use DDG::Spice;
+use URI::Escape;
+
+# Caching - https://duck.co/duckduckhack/spice_advanced_backend#caching-api-responses
+spice is_cached => 1;
+
+# Metadata.  See https://duck.co/duckduckhack/metadata for help in filling out this section.
+name "BiblioSearch - Books By Author";
```

Comment:
Please remove this Metadata, it is no longer needed in the code, the Instant Answer page contains all this info 👍